internal/api-key/command.go

```diff
@@ -38,17 +37,14 @@ const (
 	unableToStoreApiKeyErrorMsg         = "unable to store API key locally: %w"
 )
 
-func New(prerunner pcmd.PreRunner, resolver pcmd.FlagResolver) *cobra.Command {
+func New(prerunner pcmd.PreRunner) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:         "api-key",
 		Short:       "Manage API keys.",
 		Annotations: map[string]string{pcmd.RunRequirement: pcmd.RequireNonAPIKeyCloudLogin},
 	}
 
-	c := &command{
-		AuthenticatedCLICommand: pcmd.NewAuthenticatedCLICommand(cmd, prerunner),
-		flagResolver:            resolver,
-	}
+	c := &command{AuthenticatedCLICommand: pcmd.NewAuthenticatedCLICommand(cmd, prerunner)}
```

Comment:
```suggestion
	c := &command{pcmd.NewAuthenticatedCLICommand(cmd, prerunner)}
```

Reply:
Can't do this one because this command struct also has a keystore.